packages/fxa-payments-server/src/components/Header/index.tsx

```diff
@@ -1,30 +1,36 @@
 import React from 'react';
-
-import './index.scss';
 import { Profile } from '../../store/types';
+import firefoxLogo from '../../images/firefox-logo-combined.svg';
 
-type HeaderProps = {
+export type HeaderProps = {
   profile?: Profile;
   className?: string;
 };
 
 export const Header = ({ profile, className = 'default' }: HeaderProps) => {
   let profileSection = null;
+
   if (profile) {
     const { avatar, displayName, email } = profile;
     profileSection = (
       <img
-        className="avatar hoisted"
+        className="rounded-full w-8 h-8"
         src={avatar}
         data-testid="avatar"
         alt={displayName || email}
+        title={displayName || email}
```

Comment:
Added as the avatar doesn't provide a link or context.

Reply:
This feels like it should be a link... potential 1-pointer ticket?

Reply:
Agreed. Ticket filed to update the header: FXA-6233.

Since I included both notes into this PR as well as the possibility of unknowns with linking to accounts and/or signing out, I gave it 3 points. We can re-assess in triage.